driver/userspace.c

```diff
@@ -908,7 +908,7 @@ WnbdParseUserIOCTL(PWNBD_EXTENSION DeviceExtension,
             (PWNBD_IOCTL_LIST_DRV_OPT_COMMAND) Irp->AssociatedIrp.SystemBuffer;
 
         if (!ListOptCmd || CHECK_I_LOCATION(IoLocation, IOCTL_WNBD_LIST_DRV_OPT)) {
-            WNBD_LOG_ERROR("IOCTL_WNBD_RESET_DRV_OPT: Bad input or output buffer");
+            WNBD_LOG_WARN("IOCTL_WNBD_RESET_DRV_OPT: Bad input or output buffer");
```

Comment:
Let's use a debug message here. Each list command will get a warning since we're adjusting the buffer size.